frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx

```diff
@@ -71,13 +71,25 @@ const filterSelected = (list: BackingStoreStateType[], tableId: number) => {
   return list.filter((e) => e.selectedBy === tableId || e.selectedBy === '').sort(sort);
 };
 
-const getTableRows = (list: K8sResourceKind[]) => {
+const isSelectedBy = (bs, tier1, tier2) => {
+  if (tier1.length) {
+    const searchTier1 = tier1.find((elem) => elem === bs.metadata.name);
+    if (searchTier1) return 0;
+  } else if (tier2.length) {
+    const searchTier2 = tier2.find((elem) => elem === bs.metadata.name);
+    if (searchTier2) return 1;
+  }
+  return '';
```

Comment:
I will recommend returning an integer itself.
```suggestion
  return -1;
```